ergotree-ir/src/types/spreheader.rs

```diff
@@ -27,32 +29,39 @@ lazy_static! {
 }
 
 lazy_static! {
+    pub static ref VERSION_PROPERTY: SMethod = SMethod::new(PreHeader, VERSION_METHOD.clone());
```

Comment:
```suggestion
    pub static ref VERSION_PROPERTY: SMethod = SMethod::new(STypeCompanion::PreHeader, VERSION_METHOD.clone());
```
to easily distinguish between `PreHeader` type and this enum variant.